Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```diff
@@ -24,7 +24,7 @@
   name: china lake
   parent: [BaseWeaponLauncher, BaseGunWieldable, BaseSyndicateContraband]
   id: WeaponLauncherChinaLake
-  description: PLOOP.
+  description: THUMP.
```

Comment:
its THOOMP

Reply:
lol yeah I think that fits the best. The dreaded thoomp